go.mod

```diff
@@ -0,0 +1,7 @@
+module github.com/tree-sitter/tree-sitter-r
+
+go 1.23
+
+require github.com/tree-sitter/go-tree-sitter v0.23.1
+
+require github.com/mattn/go-pointer v0.0.1 // indirect
```

Comment:
What is this? It's not in the upstream code

Reply:
I'm not sure where this came from. Running `go mod tidy` added it, and without it, the Go code wouldn't build.

Reply:
It's a dependency of `tree-sitter`:

https://github.com/tree-sitter/go-tree-sitter/blob/f85230cdfd324a5943e3e905c0dd5b464d6f3409/go.mod#L6

I don't know why `go mod tidy` thinks it should be included here 🤷🏻

Reply:
Yea I see that many other grammars have it, this one is fine thanks!